data/news/platform/ocaml-5.0.beta1.md

```diff
@@ -81,15 +81,15 @@ where `option_list` is a comma separated list of `ocaml-option-*` packages. For
 ```bash
 opam switch create 5.0.0~beta1+flambda+nffa ocaml-variants.5.0.0~beta1+options ocaml-option-flambda ocaml-option-no-flat-float-array
 ```
-The command line above is slightly more complicated for opam version anterior to 2.1:
+The command line above is slightly more complicated for opam version before 2.1:
```

Comment:
I diddn't propagate the change but `opam version` should be `opam versions`, isn't it?

Reply:
Oh yes it should! Nice catch!